inst/WORDLIST

```diff
@@ -0,0 +1,66 @@
+AR
+CMD
+Chracter
+DII
+GIC
+IEA
+MWh
+NGFS
+NPVs
+PACTA
+PDs
+RStudio
+RStudio's
+Rprofile
+Tibble
+WEO
+ald
+analyze
+analyzed
```

Comment:
I assume we want British English so I used `Language: en-GB` instead of `Language: en-US` in DESCRIPTION. Feel free to change.